.github/workflows/faux-unittest.yaml

```diff
@@ -0,0 +1,16 @@
+name: faux-unittest
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+    - LICENSE
+    - '**.md'
+    - .github/workflows/faux-unittest.yaml
+    - 'docs/**'
```

Comment:
You need to include/exclude the other workflows in the `paths` and `paths-ignore`.  See https://github.com/perftool-incubator/crucible/tree/master/.github/workflows

Reply:
ack